src/core/transaction.c

```diff
@@ -657,7 +657,7 @@ static int transaction_apply(
                 /* Clean the job dependencies */
                 transaction_unlink_job(tr, j, false);
 
-                installed_job = job_install(j);
+                installed_job = job_install(j, mode);
```

Comment:
This change belongs to the wrong commit?

Reply:
Sorry I missed this one